modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -2377,11 +2379,13 @@
                   },
                   "participantClaimantId": {
                     "type": "string",
-                    "example": "123456789"
+                    "example": "123456789",
+                    "description": "Participant ID of the Claimant."
```

Comment:
I'm wondering if we need a bit more description of what type of values can be used here. I tried and ICN and I'm getting a Java error. May be worth updating our code for that but maybe that's a later ticket.

Reply:
I guess "participant ID" makes sense now that I'm thinking back. but maybe we need to look at a ticket to check these values via regex or something?